Classes/Hook/ProcessCmdmap.php

```diff
@@ -33,7 +33,7 @@ public function processCmdmap_postProcess(
         $pasteUpdate,
         $pasteDatamap
     ): void {
-        if ($command === 'delete' && $dataHandler->hasDeletedRecord($table, $id)) {
+        if ($command === 'delete') {
```

Comment:
```suggestion
        if ($command === 'delete' && \FriendsOfTYPO3\Interest\Utility\DatabaseUtility::getRecord($table, $id) === null) {
```

Not tested, but should avoid reintroducing #99.